lldb/tools/lldb-dap/LLDBUtils.cpp

```diff
@@ -252,4 +252,11 @@ std::string GetSBFileSpecPath(const lldb::SBFileSpec &file_spec) {
   return path;
 }
 
+lldb::SBLineEntry GetLineEntryForAddress(lldb::SBTarget &target,
+                                         lldb::SBAddress &address) {
+  lldb::SBSymbolContext sc =
+      address.GetSymbolContext(target, lldb::eSymbolContextLineEntry);
+  return sc.GetLineEntry();
```

Comment:
Is this the same as `address.GetLineEntry()`? Should we use that instead?

Reply:
Same reason as above, the current `SBAdress::GetLineEntry` don't apply source maps because it doesn't have the target information